build-system/pr-check/build-targets.js

```diff
@@ -98,7 +98,7 @@ const targetMatchers = [
     targets: ['INTEGRATION_TEST'],
     func: file => {
       return (
-        file.startsWith('build-system/tasks/runtime-test/') ||
+        file == 'build-system/tasks/integration.js' ||
```

Comment:
Maybe include the respective helper files too, so that tests are run if they are changed?